### PR TITLE
Fixed URI endpoints, updated configuration, and changed some data.sql

### PR DESCRIPTION
**Pull request by jaronewell**
1. The URI's in the reservation controller were fixed so that all the methods work when getting available resources.

2. The bootstrap.yml was changed to be configured for local and h2 profiles

3. Some data.sql dates were changed so that the resources would all be available during testing with the h2

### CODE REVIEW

File: src/main/java/com/revature/controllers/ReservationController.java [hunk 1]
```diff
@@ -67,12 +69,13 @@ public ReservationController(ReservationService reservationService) {
...
 	 */
 	private List<Resource> getResourcesByCampus(int campusId) {
 
 		RestTemplate restTemplate = new RestTemplate();
 		ResponseEntity<List<Resource>> response = restTemplate.exchange(
-				this.uri + "/campus/" + String.valueOf(campusId), HttpMethod.GET, null,
+				URI.create(this.uri + "/campus/" + String.valueOf(campusId)), HttpMethod.GET, null,
 				new ParameterizedTypeReference<List<Resource>>() {
 				});
 		List<Resource> resources = response.getBody();
```
[the remark this answers is not in the document]
Comment (HugoMnFernandez) [reply src/main/java/com/revature/controllers/ReservationController.java]: Makes sense, should be fixed pretty quickly

File: src/main/java/com/revature/controllers/ReservationController.java [hunk 2]
```diff
@@ -51,12 +52,13 @@ public ReservationController(ReservationService reservationService) {
...
 	 * @return A list of resources. 
+	 * @author Jaron 1811-Java-Nick 1/2/19
 	 */
 	private List<Resource> getResourcesByBuilding(int buildingId) {
-
+		
 		RestTemplate restTemplate = new RestTemplate();
 		ResponseEntity<List<Resource>> response = restTemplate.exchange(
-				this.uri + "/buildings/" + String.valueOf(buildingId), HttpMethod.GET, null,
+				URI.create(this.uri + "/building/" + String.valueOf(buildingId)), HttpMethod.GET, null,
```
[the remark this answers is not in the document]
Comment (HugoMnFernandez) [reply src/main/java/com/revature/controllers/ReservationController.java]: You don't need the "String.valueOf" statement, just 
<b> this.uri + "/buildings/" + buildingId, HttpMethod.GET, .... </b>
should be enough

File: src/main/java/com/revature/controllers/ReservationController.java [hunk 3]
```diff
@@ -67,12 +69,13 @@ public ReservationController(ReservationService reservationService) {
...
 	 * @param campusId The identification number for a campus.
 	 * @return A list of resources. 
+	 * @author Jaron 1811-Java-Nick 1/2/19
 	 */
 	private List<Resource> getResourcesByCampus(int campusId) {
 
 		RestTemplate restTemplate = new RestTemplate();
 		ResponseEntity<List<Resource>> response = restTemplate.exchange(
-				this.uri + "/campus/" + String.valueOf(campusId), HttpMethod.GET, null,
+				URI.create(this.uri + "/campus/" + String.valueOf(campusId)), HttpMethod.GET, null,
```
[the remark this answers is not in the document]
Comment (HugoMnFernandez) [reply src/main/java/com/revature/controllers/ReservationController.java]: Same as the buildingID one, and int is automatically transformed into a String when concatenated

File: src/main/java/com/revature/controllers/ReservationController.java [hunk 4]
```diff
@@ -51,12 +52,13 @@ public ReservationController(ReservationService reservationService) {
...
 	private List<Resource> getResourcesByBuilding(int buildingId) {
-
+		
 		RestTemplate restTemplate = new RestTemplate();
 		ResponseEntity<List<Resource>> response = restTemplate.exchange(
-				this.uri + "/buildings/" + String.valueOf(buildingId), HttpMethod.GET, null,
+				URI.create(this.uri + "/building/" + String.valueOf(buildingId)), HttpMethod.GET, null,
 				new ParameterizedTypeReference<List<Resource>>() {
 				});
 		List<Resource> resources = response.getBody();
```
[the remark this answers is not in the document]
Comment (HugoMnFernandez) [reply src/main/java/com/revature/controllers/ReservationController.java]: Just delete the assignment and put response.getBody in the return statement and you should be good

**Comment by HugoMnFernandez** [comment]
You should fix the sonarcloud issues before merging so that we don't add more code smells into our codebase, taking care of all of them sould take less than 10 min.

**Review by nuklearpineapple — APPROVED** [review]
Code smells and adding ReservationDto

**Pull request closed without merging**